dnsmasq/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:edge
```

Comment:
Please pin the version of the base image.

Reply:
done
`alpine:3.7`